client/index.html

```diff
@@ -16,8 +16,7 @@
 	<link rel="stylesheet" href="css/style.css">
 	<link id="theme" rel="stylesheet" href="<%= theme %>">
 
-	<link rel="shortcut icon" href="img/favicon.png">
-	<link rel="icon" sizes="192x192" href="img/touch-icon-192x192.png">
```

Comment:
This separate icon had to be removed because Chrome has some bizzare logic for selecting icons, and if you update the wrong icon, the notification badge won't be shown at all.